### ISSUE TITLE
Add not maintained sign

### ISSUE THREAD

**Issue by xhendrikg** [justdeleteme/justdelete.me#853]
Can you please write something at the top of the readme that says that this respo got moved and dont get maintained? Thanks

**Comment by tupaschoal** [comment]
I doubt anyone will, it's been inactive for 4 years now, no one ever commented about the detached fork we did :(

But it's good to have this issue here anyway to warn other people. If you're reading this, head to https://github.com/jdm-contrib/jdm !

**Comment by xhendrikg** [comment]
> I doubt anyone will, it's been inactive for 4 years now, no one ever commented about the detached fork we did :(
> 
> But it's good to have this issue here anyway to warn other people. If you're reading this, head to https://github.com/jdm-contrib/jdm !

Okay but can't you make a line above the first line with a direct Link in the readme?

**Comment by tupaschoal** [comment]
I'm not a maintainer of this repo, I just maintain the fork that was kept alive :/

**Comment by xhendrikg** [comment]
> I'm not a maintainer of this repo, I just maintain the fork that was kept alive :/

Can you message him/her?

**Comment by tupaschoal** [comment]
I cannot, I tried contacting several years ago, but had no luck

**Comment by xhendrikg** [comment]
> I cannot, I tried contacting several years ago, but had no luck

Is he/she not in the team anymore?

**Comment by tupaschoal** [comment]
I was just a fellow contributor like you are, I was never part of the team. This repository got abandoned and I decided to keep it alive on my own fork.

**Comment by xhendrikg** [comment]
> I was just a fellow contributor like you are, I was never part of the team. This repository got abandoned and I decided to keep it alive on my own fork.

So Why something New got addit it's just on GitHub and not on justdelteme.io?

**Comment by tupaschoal** [comment]
This repo is not mine. I never had any access here and can't make changes to it.

**Comment by xhendrikg** [comment]
> This repo is not mine. I never had any access here and can't make changes to it.

Yes but to the other? And does the website changes when a new "website" gets added in the sites.js?

**Comment by tupaschoal** [comment]
Yes, I maintain the other. There the website gets updated, but the website is justdeleteme.xyz

**Comment by xhendrikg** [comment]
> Yes, I maintain the other. There the website gets updated, but the website is justdeleteme.xyz

Why does the website not get down? He pays for the host but doesn't reply? And the old website is the first search result on google ...:/

**Comment by tupaschoal** [comment]
> Why does the website not get down? He pays for the host but doesn't reply? 

Yes, there are shady links there which probably get them something

> And the old website is the first search result on google ...:/

Also true, the original project received a lot of media coverage back then :/

**Comment by xhendrikg** [comment]
> > Why does the website not get down? He pays for the host but doesn't reply? 
> 
> Yes, there are shady links there which probably get them something
> 
> > And the old website is the first search result on google ...:/
> 
> Also true, the original project received a lot of media coverage back then :/

When does the website gets reloaded so the new sites are there?

**Comment by tupaschoal** [comment]
http://justdeleteme.xyz/ - as soon as the pull request is merged
https://backgroundchecks.org/justdeleteme/ - never
